### PR TITLE
Update random.py with CPython 3.7 version.

### PR DESCRIPTION
**Pull request by ChJR**
This change makes random._os, found by whats_left.sh.

### CODE REVIEW

File: Lib/random.py [hunk 1]
```diff
@@ -41,17 +41,12 @@
 from types import MethodType as _MethodType, BuiltinMethodType as _BuiltinMethodType
 from math import log as _log, exp as _exp, pi as _pi, e as _e, ceil as _ceil
 from math import sqrt as _sqrt, acos as _acos, cos as _cos, sin as _sin
-try:
-    from os import urandom as _urandom
-except ImportError:
-    # On wasm, _random.Random.random() does give a proper random value, but
-    # we don't have the os module
```
Comment (youknowone) [inline Lib/random.py:48]: Doesn wasm now have urandom?

Comment (coolreader18) [reply Lib/random.py:48]: I don't think so; @chjr could you forward-port (?) this edit to the earlier random.py version? Maybe just put the os imports in the try block?

Comment (ChJR) [reply Lib/random.py:48]: I see. I didn't check on wasm. I'll try the method suggested by @coolreader18 .

**Review by coolreader18 — APPROVED** [review]
Thanks for contributing

**Pull request merged**